# Gaelic; Scottish translation for apport
# Copyright (c) 2011 Rosetta Contributors and Canonical Ltd 2011
# This file is distributed under the same license as the apport package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2011.
#
msgid ""
msgstr ""
"Project-Id-Version: apport\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2015-09-10 11:16+0200\n"
"PO-Revision-Date: 2015-04-27 21:46+0000\n"
"Last-Translator: Martin Pitt <martin.pitt@ubuntu.com>\n"
"Language-Team: Gaelic; Scottish <gd@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Launchpad-Export-Date: 2015-12-05 05:09+0000\n"
"X-Generator: Launchpad (build 17862)\n"
"Language: \n"

#: ../debian/tmp/usr/share/apport/apport-gtk.ui.h:1
#: ../debian/tmp/usr/share/apport/apport-kde.py:468
#: ../debian/tmp/usr/share/apport/apport-kde.py:503
#: ../debian/tmp/usr/share/apport/apport-kde.py:522 ../gtk/apport-gtk.ui.h:1
#: ../kde/apport-kde.py:468 ../kde/apport-kde.py:503 ../kde/apport-kde.py:522
msgid "Apport"
msgstr ""

#: ../debian/tmp/usr/share/apport/apport-gtk.ui.h:2 ../gtk/apport-gtk.ui.h:2
msgid "Cancel"
msgstr "Sguir dheth"

#: ../debian/tmp/usr/share/apport/apport-gtk.ui.h:3 ../gtk/apport-gtk.ui.h:3
msgid "OK"
msgstr "Ceart ma-thà"

#: ../debian/tmp/usr/share/apport/apport-gtk.ui.h:4 ../gtk/apport-gtk.ui.h:4
msgid "Crash report"
msgstr "Aithris tuislidh"

#: ../debian/tmp/usr/share/apport/apport-gtk.ui.h:5 ../gtk/apport-gtk.ui.h:5
msgid "<big><b>Sorry, an internal error happened.</b></big>"
msgstr ""
"<big><b>Tha sinn duillich ach thachair mearachd inntearnail.</b></big>"

#: ../debian/tmp/usr/share/apport/apport-gtk.ui.h:6
#: ../debian/tmp/usr/share/apport/apport-kde.py:240 ../gtk/apport-gtk.ui.h:6
#: ../gtk/apport-gtk.py:300 ../debian/tmp/usr/share/apport/apport-gtk.py:300
#: ../kde/apport-kde.py:240
msgid "If you notice further problems, try restarting the computer."
msgstr ""
"Ma mhothaicheas tu do dhuilgheadasan eile, feuch is ath-thòisich an "
"coimpiutair."

#: ../debian/tmp/usr/share/apport/apport-gtk.ui.h:7 ../gtk/apport-gtk.ui.h:7
msgid "Send an error report to help fix this problem"
msgstr ""
"Cuir thugainn aithris air a' mhearachd seo gus taic a chumail rinn 'ga "
"chàraidh"

#: ../debian/tmp/usr/share/apport/apport-gtk.ui.h:8 ../gtk/apport-gtk.ui.h:8
msgid "Ignore future problems of this program version"
msgstr ""
"Leig seachad duilgheadas an tionndaidh seo dhen phrògram san àm ri teachd"

#: ../debian/tmp/usr/share/apport/apport-gtk.ui.h:9
#: ../debian/tmp/usr/share/apport/apport-kde.py:291 ../gtk/apport-gtk.ui.h:9
#: ../gtk/apport-gtk.py:204 ../gtk/apport-gtk.py:573
#: ../debian/tmp/usr/share/apport/apport-gtk.py:204
#: ../debian/tmp/usr/share/apport/apport-gtk.py:573 ../kde/apport-kde.py:291
msgid "Show Details"
msgstr "Seall am mion-fhiosrachadh"

#: ../debian/tmp/usr/share/apport/apport-gtk.ui.h:10 ../gtk/apport-gtk.ui.h:10
msgid "_Examine locally"
msgstr "_Sgrùd e gu h-ionadail"

#: ../debian/tmp/usr/share/apport/apport-gtk.ui.h:11
#: ../debian/tmp/usr/share/apport/apport-kde.py:233 ../gtk/apport-gtk.ui.h:11
#: ../gtk/apport-gtk.py:287 ../debian/tmp/usr/share/apport/apport-gtk.py:287
#: ../kde/apport-kde.py:233
msgid "Leave Closed"
msgstr "Fàg dùinte e"

#: ../debian/tmp/usr/share/apport/apport-gtk.ui.h:12
#: ../debian/tmp/usr/share/apport/apport-kde.py:230
#: ../debian/tmp/usr/share/apport/apport-kde.py:243 ../gtk/apport-gtk.ui.h:12
#: ../gtk/apport-gtk.py:216 ../gtk/apport-gtk.py:284 ../gtk/apport-gtk.py:303
#: ../debian/tmp/usr/share/apport/apport-gtk.py:216
#: ../debian/tmp/usr/share/apport/apport-gtk.py:284
#: ../debian/tmp/usr/share/apport/apport-gtk.py:303 ../kde/apport-kde.py:230
#: ../kde/apport-kde.py:243
msgid "Continue"
msgstr "Lean air adhart"

#: ../debian/tmp/usr/share/apport/apport-gtk.ui.h:13 ../gtk/apport-gtk.ui.h:13
msgid "<big><b>Collecting problem information</b></big>"
msgstr "<big><b>A' cruinneachadh fiosrachadh mun duilgheadas</b></big>"

#: ../debian/tmp/usr/share/apport/apport-gtk.ui.h:14 ../gtk/apport-gtk.ui.h:14
msgid ""
"Information is being collected that may help the developers fix the problem "
"you report."
msgstr ""
"Tha sinn a' cruinneachadh fiosrachadh a dh'fhaodadh a bhith cuideachail dhan "
"luchd-leasachaidh a' càradh an duilgheadais a rinn thu aithris air."

#: ../debian/tmp/usr/share/apport/apport-gtk.ui.h:15
#: ../debian/tmp/usr/share/apport/apport-kde.py:434 ../gtk/apport-gtk.ui.h:15
#: ../debian/tmp/usr/bin/apport-cli.py:251 ../bin/apport-cli.py:251
#: ../kde/apport-kde.py:434
msgid "Uploading problem information"
msgstr "A' luchdadh suas fiosrachadh mun duilgheadas"

#: ../debian/tmp/usr/share/apport/apport-gtk.ui.h:16 ../gtk/apport-gtk.ui.h:16
msgid "<big><b>Uploading problem information</b></big>"
msgstr "<big><b>A' luchdadh suas fiosrachadh mun duilgheadas</b></big>"

#: ../debian/tmp/usr/share/apport/apport-gtk.ui.h:17
#: ../debian/tmp/usr/share/apport/apport-kde.py:435 ../gtk/apport-gtk.ui.h:17
#: ../kde/apport-kde.py:435
msgid ""
"The collected information is being sent to the bug tracking system. This "
"might take a few minutes."
msgstr ""
"Tha am fiosrachadh a chaidh a chruinneachadh ga chur gu siostam tracadh nam "
"bugaichean. Dh'fhaoidte gum feum seo beagan mhionaidean."

#: ../kde/apport-kde-mimelnk.desktop.in.h:1
msgid "Apport crash file"
msgstr "Faidhle tuislidh  Apport"

#: ../debian/tmp/usr/share/apport/apport-kde.py:362 ../gtk/apport-gtk.py:142
#: ../debian/tmp/usr/share/apport/apport-gtk.py:142
#: ../debian/tmp/usr/bin/apport-cli.py:150 ../bin/apport-cli.py:150
#: ../kde/apport-kde.py:362
msgid "(binary data)"
msgstr "(dàta bìnearaidh)"

#: ../gtk/apport-gtk.py:157 ../debian/tmp/usr/share/apport/apport-gtk.py:157
#, python-format
msgid "Sorry, the application %s has stopped unexpectedly."
msgstr "Tha sinn duilich ach stad an aplacaid %s gu h-obann."

#: ../gtk/apport-gtk.py:160 ../debian/tmp/usr/share/apport/apport-gtk.py:160
#, python-format
msgid "Sorry, %s has closed unexpectedly."
msgstr "Tha sinn duilich ach dhùin %s  gu h-obann."

#: ../debian/tmp/usr/share/apport/apport-kde.py:199
#: ../debian/tmp/usr/share/apport/apport-kde.py:237 ../gtk/apport-gtk.py:165
#: ../debian/tmp/usr/share/apport/apport-gtk.py:165 ../kde/apport-kde.py:199
#: ../kde/apport-kde.py:237
#, python-format
msgid "Sorry, %s has experienced an internal error."
msgstr "Tha sinn duilich, thachair mearachd inntearnail ann an %s."

#: ../debian/tmp/usr/share/apport/apport-kde.py:185 ../gtk/apport-gtk.py:177
#: ../debian/tmp/usr/share/apport/apport-gtk.py:177
#: ../debian/tmp/usr/bin/apport-cli.py:178 ../bin/apport-cli.py:178
#: ../kde/apport-kde.py:185
msgid "Send problem report to the developers?"
msgstr ""
"A bheil thu airson aithris mun duilgheadas a chur gun luchd-leasachaidh?"

#: ../debian/tmp/usr/share/apport/apport-kde.py:193 ../gtk/apport-gtk.py:186
#: ../debian/tmp/usr/share/apport/apport-gtk.py:186 ../kde/apport-kde.py:193
msgid "Send"
msgstr ""

#: ../gtk/apport-gtk.py:228 ../debian/tmp/usr/share/apport/apport-gtk.py:228
msgid "Force Closed"
msgstr "Sparr air dùnadh"

#: ../debian/tmp/usr/share/apport/apport-kde.py:234
#: ../debian/tmp/usr/share/apport/apport-kde.py:380 ../gtk/apport-gtk.py:229
#: ../gtk/apport-gtk.py:288 ../debian/tmp/usr/share/apport/apport-gtk.py:229
#: ../debian/tmp/usr/share/apport/apport-gtk.py:288 ../kde/apport-kde.py:234
#: ../kde/apport-kde.py:380
msgid "Relaunch"
msgstr ""

#: ../gtk/apport-gtk.py:238 ../debian/tmp/usr/share/apport/apport-gtk.py:238
#, python-format
msgid "The application %s has stopped responding."
msgstr "Chan eil an aplacaid \"%s\" a' freagairt tuilleadh."

#: ../gtk/apport-gtk.py:242 ../debian/tmp/usr/share/apport/apport-gtk.py:242
#, python-format
msgid "The program \"%s\" has stopped responding."
msgstr "Chan eil am prògram \"%s\" a' freagairt tuilleadh."

#: ../debian/tmp/usr/share/apport/apport-kde.py:207 ../gtk/apport-gtk.py:257
#: ../debian/tmp/usr/share/apport/apport-gtk.py:257 ../kde/apport-kde.py:207
#, python-format
msgid "Package: %s"
msgstr ""

#: ../debian/tmp/usr/share/apport/apport-kde.py:213 ../gtk/apport-gtk.py:264
#: ../debian/tmp/usr/share/apport/apport-gtk.py:264 ../kde/apport-kde.py:213
msgid "Sorry, a problem occurred while installing software."
msgstr ""
"Tha sinn duilich, dh'èirich duilgheadas nuair a bha sinn a' stàladh a' "
"bhathair-bhog."

#: ../debian/tmp/usr/share/apport/apport-kde.py:219 ../gtk/apport-gtk.py:273
#: ../gtk/apport-gtk.py:292 ../debian/tmp/usr/share/apport/apport-gtk.py:273
#: ../debian/tmp/usr/share/apport/apport-gtk.py:292 ../kde/apport-kde.py:219
#, python-format
msgid "The application %s has experienced an internal error."
msgstr "Thachair an aplacaid %s ri mearachd inntearnail."

#: ../debian/tmp/usr/share/apport/apport-kde.py:222 ../gtk/apport-gtk.py:276
#: ../debian/tmp/usr/share/apport/apport-gtk.py:276 ../kde/apport-kde.py:222
#, python-format
msgid "The application %s has closed unexpectedly."
msgstr "Dhùin an aplacaid %s gu h-obann."

#: ../debian/tmp/usr/share/apport/apport-kde.py:244 ../gtk/apport-gtk.py:304
#: ../debian/tmp/usr/share/apport/apport-gtk.py:304 ../kde/apport-kde.py:244
msgid "Ignore future problems of this type"
msgstr "Leig seachad duilgheadasan dhen t-seòrsa seo san àm ri teachd"

#: ../debian/tmp/usr/share/apport/apport-kde.py:288 ../gtk/apport-gtk.py:577
#: ../debian/tmp/usr/share/apport/apport-gtk.py:577 ../kde/apport-kde.py:288
msgid "Hide Details"
msgstr "Falaich am mion-fhiosrachadh"

#: ../bin/apport-unpack.py:22 ../debian/tmp/usr/bin/apport-unpack.py:22
#, python-format
msgid "Usage: %s <report> <target directory>"
msgstr "Pàtran: %s <report> <target directory>"

#: ../bin/apport-unpack.py:46 ../debian/tmp/usr/bin/apport-unpack.py:46
msgid "Destination directory exists and is not empty."
msgstr "Tha am pasgan sin sa cheann-uidhe mu thràth 's chan eil e falamh."

#: ../debian/tmp/usr/share/apport/apport-kde.py:314 ../kde/apport-kde.py:314
msgid "Username:"
msgstr ""

#: ../debian/tmp/usr/share/apport/apport-kde.py:315 ../kde/apport-kde.py:315
msgid "Password:"
msgstr ""

#: ../debian/tmp/usr/share/apport/apport-kde.py:405 ../kde/apport-kde.py:405
msgid "Collecting Problem Information"
msgstr "A' cruinneachadh fiosrachadh mun duilgheadas"

#: ../debian/tmp/usr/share/apport/apport-kde.py:406
#: ../debian/tmp/usr/bin/apport-cli.py:238 ../bin/apport-cli.py:238
#: ../kde/apport-kde.py:406
msgid "Collecting problem information"
msgstr "A' cruinneachadh fiosrachadh mun duilgheadas"

#: ../debian/tmp/usr/share/apport/apport-kde.py:407 ../kde/apport-kde.py:407
msgid ""
"The collected information can be sent to the developers to improve the "
"application. This might take a few minutes."
msgstr ""
"'S urrainn dhut am fiosrachadh a chaidh a chruinneachadh a chur dhan luchd-"
"leasachaidh airson 's gun urrainn dhaibh an aplacaid a leasachadh. "
"Dh'fhaoidte gum feum seo beagan mhionaidean."

#: ../debian/tmp/usr/share/apport/apport-kde.py:433 ../kde/apport-kde.py:433
msgid "Uploading Problem Information"
msgstr "A' luchdadh suas fiosrachadh mun duilgheadas"

#: ../apport/com.ubuntu.apport.policy.in.h:1
msgid "Collect system information"
msgstr "Cruinnich fiosrachaidh mun t-siostam"

#: ../apport/com.ubuntu.apport.policy.in.h:2
msgid ""
"Authentication is required to collect system information for this problem "
"report"
msgstr ""
"Tha feum air dearbhadh mus urrainn dhuinn fiosrachadh a chruinneachadh mun t-"
"siostam airson na h-aithrise seo"

#: ../apport/com.ubuntu.apport.policy.in.h:3
msgid "System problem reports"
msgstr "Aithrisean air duilgheadasan an t-siostaim"

#: ../apport/com.ubuntu.apport.policy.in.h:4
msgid ""
"Please enter your password to access problem reports of system programs"
msgstr ""
"Cuir a-steach am facal-faire agad gus cothrom fhaighinn air aithrisean mu "
"dhèidhinn duilgheadas a bha aig prògraman a t-siostaim"

#: ../kde/apport-kde-mime.desktop.in.h:1 ../kde/apport-kde.desktop.in.h:1
#: ../gtk/apport-gtk.desktop.in.h:1
msgid "Report a problem..."
msgstr "Dèan aithris air duilgheadas..."

#: ../kde/apport-kde-mime.desktop.in.h:2 ../kde/apport-kde.desktop.in.h:2
#: ../gtk/apport-gtk.desktop.in.h:2
msgid "Report a malfunction to the developers"
msgstr "Dèan aithris air mearachd dhan luchd-leasachaidh"

#: ../debian/tmp/usr/bin/apport-valgrind.py:37 ../bin/apport-valgrind.py:37
msgid "See man page for details."
msgstr "Faic duilleag na treòire airson mion-fhiosrachadh."

#: ../debian/tmp/usr/bin/apport-valgrind.py:43 ../bin/apport-valgrind.py:43
msgid "specify the log file name produced by valgrind"
msgstr "sònraich ainm faidhle an loga a rinn valgrind"

#: ../debian/tmp/usr/bin/apport-valgrind.py:46 ../bin/apport-valgrind.py:46
msgid ""
"reuse a previously created sandbox dir (SDIR) or, if it does not exist, "
"create it"
msgstr ""
"Cleachd Sandbox Dir (SDIR) a-rithist a chaidh a chruthachadh roimhe no, mur "
"eil e ann, cruthaich e"

#: ../debian/tmp/usr/bin/apport-valgrind.py:50 ../bin/apport-valgrind.py:50
msgid ""
"do  not  create  or reuse a sandbox directory for additional debug symbols "
"but rely only on installed debug symbols."
msgstr ""
"na cruthaich 's na cleachd Sandbox Directory a-rithist airson samhlaidhean "
"dì-bhugachaidh a bharrachd is cleachd dìreach an fheadhainn a tha stàlaichte."

#: ../debian/tmp/usr/bin/apport-valgrind.py:54 ../bin/apport-valgrind.py:54
msgid ""
"reuse a previously created cache dir (CDIR) or, if it does not exist, create "
"it"
msgstr ""
"Cleachd Cache Dir (CDIR) a-rithist a chaidh a chruthachadh roimhe no, mur "
"eil e ann, cruthaich e"

#: ../debian/tmp/usr/bin/apport-valgrind.py:58 ../bin/apport-valgrind.py:58
msgid ""
"report download/install progress when installing packages into sandbox"
msgstr ""
"seall adhartas an luchdaidh a-nuas/an stàlaidh nuair a stàlaichear pacaidean "
"sa bhogsa-ghainmhich"

#: ../debian/tmp/usr/bin/apport-valgrind.py:62 ../bin/apport-valgrind.py:62
msgid ""
"the executable that is run under valgrind's memcheck tool  for memory leak "
"detection"
msgstr ""
"an executable a thèid a ruith fo inneal sgrùdadh na cuimhne aig valgrind "
"airson greim fhaighinn air aoidion cuimhne"

#: ../debian/tmp/usr/bin/apport-valgrind.py:66 ../bin/apport-retrace.py:63
#: ../bin/apport-valgrind.py:66 ../debian/tmp/usr/bin/apport-retrace.py:63
msgid ""
"Install an extra package into the sandbox (can be specified multiple times)"
msgstr ""
"Stàlaich pacaid a bharrachd san bhogsa-ghainmhich (gabhaidh seo a "
"shònrachadh iomadh turas)"

#: ../debian/tmp/usr/bin/apport-valgrind.py:97 ../bin/apport-valgrind.py:97
#, python-format
msgid "Error: %s is not an executable. Stopping."
msgstr "Mearachd: Chan eil %s 'na rud so-ghnìomhaichte. A' sgur dheth."

#: ../data/kernel_oops.py:29 ../debian/tmp/usr/share/apport/kernel_oops.py:29
msgid "Your system might become unstable now and might need to be restarted."
msgstr ""
"Dh'fhaoidte gum fàs an siostam agad neo-sheasmhach a-nis is gum b' fheairrde "
"dhut ath-thòiseachadh."

#: ../bin/apport-retrace.py:34 ../debian/tmp/usr/bin/apport-retrace.py:34
msgid "Do not put the new traces into the report, but write them to stdout."
msgstr "Na cuir na traces ùra dhan aithris ach sgrìobh iad ann an stdout."

#: ../bin/apport-retrace.py:36 ../debian/tmp/usr/bin/apport-retrace.py:36
msgid ""
"Start an interactive gdb session with the report's core dump (-o ignored; "
"does not rewrite report)"
msgstr ""
"Tòisich air seisean gdb eadar-ghnìomhach le core dump na h-aithrise (-o air "
"a leigeil seachad; cha dèan e ath-sgrìobhadh na h-aithrise)"

#: ../bin/apport-retrace.py:38 ../debian/tmp/usr/bin/apport-retrace.py:38
msgid ""
"Write modified report to given file instead of changing the original report"
msgstr ""
"Sgrìobh an aithris atharraichte san fhaidhle a tha ann 's na atharraich an "
"aithris thùsail"

#: ../bin/apport-retrace.py:41 ../debian/tmp/usr/bin/apport-retrace.py:41
msgid "Remove the core dump from the report after stack trace regeneration"
msgstr ""
"Thoir air falbh an core dump às an aithris às dèidh ath-ghintinn an stack "
"trace"

#: ../bin/apport-retrace.py:43 ../debian/tmp/usr/bin/apport-retrace.py:43
msgid "Override report's CoreFile"
msgstr "Tar-àithn CoreFile na h-aithrise"

#: ../bin/apport-retrace.py:45 ../debian/tmp/usr/bin/apport-retrace.py:45
msgid "Override report's ExecutablePath"
msgstr "Tar-àithn ExecutablePath na h-aithrise"

#: ../bin/apport-retrace.py:47 ../debian/tmp/usr/bin/apport-retrace.py:47
msgid "Override report's ProcMaps"
msgstr "Tar-àithn ProcMaps na h-aithrise"

#: ../bin/apport-retrace.py:49 ../debian/tmp/usr/bin/apport-retrace.py:49
msgid "Rebuild report's Package information"
msgstr "Ath-thog fiosrachadh pacaid na h-aithrise"

#: ../bin/apport-retrace.py:51 ../debian/tmp/usr/bin/apport-retrace.py:51
msgid ""
"Build a temporary sandbox and download/install the necessary packages and "
"debug symbols in there; without this option it assumes that the necessary "
"packages and debug symbols are already installed in the system. The argument "
"points to the packaging system configuration base directory; if you specify "
"\"system\", it will use the system configuration files, but will then only "
"be able to retrace crashes that happened on the currently running release."
msgstr ""
"Tog bogsa-gainmhich sealach is luchdaich a-nuas/stàlaich na pacaidean is "
"samhlaidhean dì-bhugachaidh air a bheil feum ann; as aonais na roghainn seo, "
"tuigear dheth gun deach na pacaidean is samhlaidhean dì-bhugachaidh "
"iomchaidh a stàladh air an t-siostam mu thràth. Tha an argamaid seo a' "
"tomhadh ris a' bhun-phasgan aig rèiteachadh siostam nam pacaidean; ma "
"shònraicheas tu \"system\", cleachdaidh e faidhlichean rèiteachaidh an t-"
"siostaim ach ma nì thu sin, chan urrainn dha retrace a dhèanamh air "
"tuislidhean ach feadhainn san sgaoileadh a tha a' ruith an-dràsta fhèin."

#: ../bin/apport-retrace.py:53 ../debian/tmp/usr/bin/apport-retrace.py:53
msgid ""
"Report download/install progress when installing packages into sandbox"
msgstr ""
"Seall adhartas an luchdaidh a-nuas/an stàlaidh nuair a stàlaichear pacaidean "
"sa bhogsa-ghainmhich"

#: ../bin/apport-retrace.py:55 ../debian/tmp/usr/bin/apport-retrace.py:55
msgid "Prepend timestamps to log messages, for batch operation"
msgstr ""
"Cuir stampa-ama air beulaibh teachdaireachdan an loga, airson gnìomhachadh "
"batch"

#: ../bin/apport-retrace.py:57 ../debian/tmp/usr/bin/apport-retrace.py:57
msgid ""
"Create and use third-party repositories from origins specified in reports"
msgstr ""

#: ../bin/apport-retrace.py:59 ../debian/tmp/usr/bin/apport-retrace.py:59
msgid "Cache directory for packages downloaded in the sandbox"
msgstr ""
"Pasgan an tasgadain airson pacaidean a chaidh a luchdadh a-nuas dhan bhogsa-"
"ghainmhich"

#: ../bin/apport-retrace.py:61 ../debian/tmp/usr/bin/apport-retrace.py:61
msgid ""
"Directory for unpacked packages. Future runs will assume that any already "
"downloaded package is also extracted to this sandbox."
msgstr ""
"Am pasgan airson pacaidean a chaidh fhosgladh. Tuigear dheth ann an ruith "
"sam bith o seo a-mach gun dèid pacaid sam bith a chaidh a luchdadh a-nuas "
"fhosgladh 's a chur dhan bhogsa-ghainmhich seo mu thràth."

#: ../bin/apport-retrace.py:65 ../debian/tmp/usr/bin/apport-retrace.py:65
msgid ""
"Path to a file with the crash database authentication information. This is "
"used when specifying a crash ID to upload the retraced stack traces (only if "
"neither -g, -o, nor -s are specified)"
msgstr ""
"Slighe gu faidhle sa bheil am fiosrachadh airson dearbhadh aig stòr-dàta nan "
"tuislidhean. Tha seo 'ga chleachdadh nuair a shònraichear ID tuislidh gus na "
"retraced stack traces a luchdadh suas (dìreach mur an deach -g, -o no -s a "
"shònrachadh)"

#: ../bin/apport-retrace.py:67 ../debian/tmp/usr/bin/apport-retrace.py:67
msgid ""
"Display retraced stack traces and ask for confirmation before sending them "
"to the crash database."
msgstr ""
"Seall na retraced stack traces is faighnich dhìom mus dèid an cur gu stòr-"
"dàta nan tuislidhean."

#: ../bin/apport-retrace.py:69 ../debian/tmp/usr/bin/apport-retrace.py:69
msgid ""
"Path to the duplicate sqlite database (default: no duplicate checking)"
msgstr ""
"An t-slighe gun stòr-dàta sqlite dùblaichte (bun-roghainn: cha doirear sùil "
"a bheil iad dùblaichte)"

#: ../bin/apport-retrace.py:78 ../debian/tmp/usr/bin/apport-retrace.py:78
msgid "You cannot use -C without -S. Stopping."
msgstr "Chan urrainn dhut -C a chleachdadh às aonais -S. A' stad."

#. translators: don't translate y/n, apport currently only checks for "y"
#: ../bin/apport-retrace.py:111 ../debian/tmp/usr/bin/apport-retrace.py:111
msgid "OK to send these as attachments? [y/n]"
msgstr "A bheil e ceart gu leòr an cur mar cheanglachain? [y/n]"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:124
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:124
#: ../apport/ui.py:124
msgid "This package does not seem to be installed correctly"
msgstr "Tha coltas nach deach a' phacaid seo a stàladh mar bu chòir"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:129
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:129
#: ../apport/ui.py:129
#, python-format
msgid ""
"This is not an official %s package. Please remove any third party package "
"and try again."
msgstr ""
"Chan e  pacaid %s oifigeil a tha seo. Thoir air falbh pacaid sam bith a tha "
"le treas-phàrtaidh 's feuch ris a-rithist."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:146
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:146
#: ../apport/ui.py:146
#, python-format
msgid ""
"You have some obsolete package versions installed. Please upgrade the "
"following packages and check if the problem still occurs:\n"
"\n"
"%s"
msgstr ""
"Tha cuid a phacaidean stàlaichte agad a tha ro aosta. Àrdaich na pacaidean a "
"leanas agus thoir sùil ach a bheil an duilgheadas ann fhathast:\n"
"\n"
" %s"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:270
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:270
#: ../apport/ui.py:270
msgid "unknown program"
msgstr ""

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:271
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:271
#: ../apport/ui.py:271
#, python-format
msgid "Sorry, the program \"%s\" closed unexpectedly"
msgstr "Tha sinn duilich ach dhùin am prògram \"%s\" gu h-obann."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:273
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1312
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:273
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:1312
#: ../apport/ui.py:273 ../apport/ui.py:1312
#, python-format
msgid "Problem in %s"
msgstr "Duilgheadas ann an %s"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:274
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:274
#: ../apport/ui.py:274
msgid ""
"Your computer does not have enough free memory to automatically analyze the "
"problem and send a report to the developers."
msgstr ""
"Chan eil cuimhne shaor gu leòr aig a' choimpiutair agad agus chan urrainn "
"dha an duilgheadas a sgrùdadh gu fèin-obrachail is aithris a chur dhan luchd-"
"leasachaidh."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:322
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:330
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:457
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:460
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:661
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1118
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1284
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1288
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:322
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:330
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:457
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:460
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:661
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:1118
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:1284
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:1288
#: ../apport/ui.py:322 ../apport/ui.py:330 ../apport/ui.py:457
#: ../apport/ui.py:460 ../apport/ui.py:661 ../apport/ui.py:1118
#: ../apport/ui.py:1284 ../apport/ui.py:1288
msgid "Invalid problem report"
msgstr "Aithris duilgheadais mhì-dhligheach"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:323
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:323
#: ../apport/ui.py:323
msgid "You are not allowed to access this problem report."
msgstr "Chan eil cead-inntrigidh agad dhan aithris duilgheadais seo."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:326
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:326
#: ../apport/ui.py:326
msgid "Error"
msgstr ""

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:327
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:327
#: ../apport/ui.py:327
msgid "There is not enough disk space available to process this report."
msgstr ""
"Chan eil àite gu leòr air an diosg gus an aithris seo a làimhseachadh."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:411
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:411
#: ../apport/ui.py:411
msgid "No package specified"
msgstr ""

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:412
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:412
#: ../apport/ui.py:412
msgid ""
"You need to specify a package or a PID. See --help for more information."
msgstr ""
"Feumaidh tu pacaid no PID a shònrachadh. Faic --help airson barrachd "
"fiosrachaidh."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:435
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:435
#: ../apport/ui.py:435
msgid "Permission denied"
msgstr "Chaidh cead a dhiùltadh"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:436
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:436
#: ../apport/ui.py:436
msgid ""
"The specified process does not belong to you. Please run this program as the "
"process owner or as root."
msgstr ""
"Chan eil am pròiseas a shònraich thu a' buntainn riut. Ruith am prògram seo "
"mar shealbhadair a' phròiseis no mar root."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:438
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:438
#: ../apport/ui.py:438
msgid "Invalid PID"
msgstr "PID mì-dhligheach"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:439
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:439
#: ../apport/ui.py:439
msgid "The specified process ID does not belong to a program."
msgstr "Chan eil ID a' phròiseis a shònraich thu a' buntainn do phrògram."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:458
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:458
#: ../apport/ui.py:458
#, python-format
msgid "Symptom script %s did not determine an affected package"
msgstr "Cha do lorg an sgriobt %s pacaid air a bheil buaidh"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:461
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:461
#: ../apport/ui.py:461
#, python-format
msgid "Package %s does not exist"
msgstr "Chan eil a' phacaid %s ann"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:485
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:673
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:678
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:485
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:673
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:678
#: ../apport/ui.py:485 ../apport/ui.py:673 ../apport/ui.py:678
msgid "Cannot create report"
msgstr "Cha ghabh an aithris a chruthachadh"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:500
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:546
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:563
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:500
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:546
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:563
#: ../apport/ui.py:500 ../apport/ui.py:546 ../apport/ui.py:563
msgid "Updating problem report"
msgstr "Ag ùrachadh aithris an duilgheadais"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:501
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:501
#: ../apport/ui.py:501
msgid ""
"You are not the reporter or subscriber of this problem report, or the report "
"is a duplicate or already closed.\n"
"\n"
"Please create a new report using \"apport-bug\"."
msgstr ""
"Cha tusa an neach a rinn aithris air an duilgheadas seo 's chan eil fo-"
"sgrìobhadh agad ann air neo bha an aithris seo ann dà thuras no chaidh a "
"dhùnadh mu thràth.\n"
"\n"
"Cruthaich aithris ùr le \"apport-bug\"."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:510
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:510
#: ../apport/ui.py:510
msgid ""
"You are not the reporter of this problem report. It is much easier to mark a "
"bug as a duplicate of another than to move your comments and attachments to "
"a new bug.\n"
"\n"
"Subsequently, we recommend that you file a new bug report using \"apport-"
"bug\" and make a comment in this bug about the one you file.\n"
"\n"
"Do you really want to proceed?"
msgstr ""
"Cha tusa an neach a rinn aithris air an duilgheadas seo. Tha e fada nas "
"fhasa comharra a chur ri buga a dh'innseas gur e dùblachadh a tha ann seach "
"na beachdan is ceanglachain agad a ghluasad gu buga ùr.\n"
"\n"
"Leis a sin, mholamaid dhut aithris ùr a dhèanamh air a' bhuga le \"apport-"
"bug\" agus beachd a chur ris a' bhuga seo mun fhear a chlàraich thu.\n"
"\n"
"A bheil thu cinnteach gu bheil thu airson leantainn air adhart?"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:547
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:564
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:547
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:564
#: ../apport/ui.py:547 ../apport/ui.py:564
msgid "No additional information collected."
msgstr "Cha deach fiosrachadh a bharrachd a chruinneachadh."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:615
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:615
#: ../apport/ui.py:615
msgid "What kind of problem do you want to report?"
msgstr "Dè seòrsa duilgheadas tha thu airson aithris a dhèanamh air?"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:632
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:632
#: ../apport/ui.py:632
msgid "Unknown symptom"
msgstr "Buaidh nach aithne dhuinn"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:633
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:633
#: ../apport/ui.py:633
#, python-format
msgid "The symptom \"%s\" is not known."
msgstr "Chan aithne dhuinn a' bhuaidh \"%s\"."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:664
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:664
#: ../apport/ui.py:664
msgid ""
"After closing this message please click on an application window to report a "
"problem about it."
msgstr ""
"An dèidh dhut an teachdaireachd seo a dhùnadh. briog air uinneag aplacaid "
"gus aithris a dhèanamh air duilgheadas leis."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:674
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:679
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:674
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:679
#: ../apport/ui.py:674 ../apport/ui.py:679
msgid "xprop failed to determine process ID of the window"
msgstr "Cha b' urrainn dha xprop ID pròiseas na h-uinneige a dhearbhadh"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:693
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:693
#: ../apport/ui.py:693
msgid "%prog <report number>"
msgstr ""

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:695
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:695
#: ../apport/ui.py:695
msgid "Specify package name."
msgstr "Sònraich ainm na pacaid."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:697
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:748
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:697
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:748
#: ../apport/ui.py:697 ../apport/ui.py:748
msgid "Add an extra tag to the report. Can be specified multiple times."
msgstr ""
"Cuir taga a bharrachd ris an aithris. 'S urrainn dhut a shònrachadh iomadh "
"turas."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:727
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:727
#: ../apport/ui.py:727
msgid ""
"%prog [options] [symptom|pid|package|program path|.apport/.crash file]"
msgstr ""
"%prog [options] [symptom|pid|package|program path|.apport/.crash file]"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:730
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:730
#: ../apport/ui.py:730
msgid ""
"Start in bug filing mode. Requires --package and an optional --pid, or just "
"a --pid. If neither is given, display a list of known symptoms. (Implied if "
"a single argument is given.)"
msgstr ""
"Tòisich sa mhodh aithriseadh bhugaichean. Feumaidh seo --package agus --pid "
"roghainneil no dìreach --pid. Mur eil aon seach aon dhiubh ann, nochd liosta "
"de bhuaidhean air a bheil eòlas. (Tuigear dheth seo ma tha argamaid "
"shingilte air a chur ann)."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:732
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:732
#: ../apport/ui.py:732
msgid "Click a window as a target for filing a problem report."
msgstr ""
"Briog air uinneag mar thargaid gus aithris a dhèanamh air duilgheadas."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:734
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:734
#: ../apport/ui.py:734
msgid "Start in bug updating mode. Can take an optional --package."
msgstr ""
"Tòisich sa mhodh ùrachadh bhugaichean. Gabhaidh seo ri --package ma tha feum "
"air."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:736
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:736
#: ../apport/ui.py:736
msgid ""
"File a bug report about a symptom. (Implied if symptom name is given as only "
"argument.)"
msgstr ""
"Clàraich buga mu dhèidhinn buaidh. (Tuigear dheth seo mas e ainm na buaidhe "
"an aon argamaid.)"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:738
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:738
#: ../apport/ui.py:738
msgid ""
"Specify package name in --file-bug mode. This is optional if a --pid is "
"specified. (Implied if package name is given as only argument.)"
msgstr ""
"Sònraich ainm na pacaid sa mhodh --file-bug. Tha seo roghainneil ma chaidh --"
"pid a shònrachadh. (Tuigear dheth seo mas e ainm na pacaid an aon argamaid.)"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:740
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:740
#: ../apport/ui.py:740
msgid ""
"Specify a running program in --file-bug mode. If this is specified, the bug "
"report will contain more information.  (Implied if pid is given as only "
"argument.)"
msgstr ""
"Sònraich prògram a tha a' ruith sa mhodh --file-bug. Ma thèid seo a "
"shònrachadh, bidh barrachd fiosrachaidh san aithris. (Tuigear dheth seo mas "
"e pid an aon argamaid.)"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:742
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:742
#: ../apport/ui.py:742
msgid "The provided pid is a hanging application."
msgstr "Tha am pid a shònraich thu 'na aplacaid chrochte."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:744
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:744
#: ../apport/ui.py:744
#, python-format
msgid ""
"Report the crash from given .apport or .crash file instead of the pending "
"ones in %s. (Implied if file is given as only argument.)"
msgstr ""
"Dèan aithris air an tuisleadh on fhaidhle .apport no .crash a tha ann seach "
"an fheadhainn a tha ri dhèiligeadh ann an %s. (Tuigear dheth seo mas e am "
"faidhle an aon argamaid.)"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:746
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:746
#: ../apport/ui.py:746
msgid ""
"In bug filing mode, save the collected information into a file instead of "
"reporting it. This file can then be reported later on from a different "
"machine."
msgstr ""
"Sa mhodh aithriseadh bhugaichean, sàbhail am fiosrachadh a chaidh a "
"chruinneachadh ann am faidhle seach a bhith a' dèanamh aithris air. 'S "
"urrainn dhut aithris a dhèanamh leis an fhaidhle às a dhèidh sin no o "
"choimpiutair eile."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:750
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:750
#: ../apport/ui.py:750
msgid "Print the Apport version number."
msgstr "Clò-bhuail àireamh an tionndaidh aig Apport."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:889
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:889
#: ../apport/ui.py:889
msgid ""
"This will launch apport-retrace in a terminal window to examine the crash."
msgstr ""
"Cuireadh seo air bhog apport-retrace ann an uinneag tèirmineil gus sgrùdadh "
"a dhèanamh air an tuisleadh."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:890
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:890
#: ../apport/ui.py:890
msgid "Run gdb session"
msgstr ""

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:891
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:891
#: ../apport/ui.py:891
msgid "Run gdb session without downloading debug symbols"
msgstr ""
"Ruith seisean gdb as aonais gun a bhith a' luchdadh a-nuas nan samhlaidhean "
"dì-bhugachaidh"

#. TRANSLATORS: %s contains the crash report file name
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:893
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:893
#: ../apport/ui.py:893
#, python-format
msgid "Update %s with fully symbolic stack trace"
msgstr "Ùraich %s le Stack Trace a tha gu tur samhlach"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:969
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:979
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:969
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:979
#: ../apport/ui.py:969 ../apport/ui.py:979
msgid ""
"This problem report applies to a program which is not installed any more."
msgstr ""
"Tha aithris an duilgheadais seo a' buntainn ri prògram nach eil stàlaichte "
"tuilleadh."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:994
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:994
#: ../apport/ui.py:994
#, python-format
msgid ""
"The problem happened with the program %s which changed since the crash "
"occurred."
msgstr ""
"Thachair an duilgheadas leis a' phrògram %s ach chaidh atharrachadh on a "
"thachair an tuisleadh."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1041
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1075
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1290
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:1041
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:1075
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:1290
#: ../apport/ui.py:1041 ../apport/ui.py:1075 ../apport/ui.py:1290
msgid "This problem report is damaged and cannot be processed."
msgstr "Chaidh an aithris seo a dhochann is cha ghabh a làimhseachadh."

#. package does not exist
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1045
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:1045
#: ../apport/ui.py:1045
msgid "The report belongs to a package that is not installed."
msgstr "Tha an aithris seo a' buntainn do phacaid nach eil stàlaichte."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1049
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:1049
#: ../apport/ui.py:1049
msgid "An error occurred while attempting to process this problem report:"
msgstr ""
"Thachair mearachd fhad 's a bha sinn a' feuchainn ri aithris na mearachd seo "
"a làimhseachadh:"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1119
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:1119
#: ../apport/ui.py:1119
msgid "Could not determine the package or source package name."
msgstr ""
"Cha b' urrainn dhuinn a' phacaid no ainm na pacaid thùsail a dhearbhadh."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1137
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:1137
#: ../apport/ui.py:1137
msgid "Unable to start web browser"
msgstr "Chan urrainn dhuinn brabhsair-lìn a thòiseachadh"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1138
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:1138
#: ../apport/ui.py:1138
#, python-format
msgid "Unable to start web browser to open %s."
msgstr "Chan urrainn dhuinn brabhsair-lìn a thòiseachadh gus %s fhosgladh."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1238
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:1238
#: ../apport/ui.py:1238
#, python-format
msgid "Please enter your account information for the %s bug tracking system"
msgstr ""
"Cuir a-steach fiosrachadh a' chunntais agad airson siostam tracadh nam "
"bugaichean aig %s"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1250
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:1250
#: ../apport/ui.py:1250
msgid "Network problem"
msgstr "Duilgheadas leis an lìonra"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1252
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:1252
#: ../apport/ui.py:1252
msgid ""
"Cannot connect to crash database, please check your Internet connection."
msgstr ""
"Chan urrainn dhuinn ceangal ri stòr-dàta nan tuisleadh, thoir sùil air a' "
"cheangal agad ris an eadar-lìon."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1279
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:1279
#: ../apport/ui.py:1279
msgid "Memory exhaustion"
msgstr "Gainnead cuimhne"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1280
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:1280
#: ../apport/ui.py:1280
msgid "Your system does not have enough memory to process this crash report."
msgstr ""
"Chan eil cuimhne gu leòr air an t-siostam agad is chan urrainn dha aithris "
"an tuislidh seo a làimhseachadh."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1315
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:1315
#: ../apport/ui.py:1315
#, python-format
msgid ""
"The problem cannot be reported:\n"
"\n"
"%s"
msgstr ""
"Chan urrainn dhuinn aithris a dhèanamh air an duilgheadas:\n"
"\n"
"%s"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1371
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1378
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:1371
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:1378
#: ../apport/ui.py:1371 ../apport/ui.py:1378
msgid "Problem already known"
msgstr "Tha sinn eòlach air an duilgheadas seo mu thràth"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1372
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:1372
#: ../apport/ui.py:1372
msgid ""
"This problem was already reported in the bug report displayed in the web "
"browser. Please check if you can add any further information that might be "
"helpful for the developers."
msgstr ""
"Chaidh aithris a dhèanamh air an duilgheadas seo mu thràth is chì thu "
"aithris a' bhuga sa bhrabhsair-lìn agad. Thoir sùil ach a bheil fiosrachadh "
"ùr sam bith agad a dh'fhaodadh a bhith feumail dhan luchd-leasachaidh."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1379
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:1379
#: ../apport/ui.py:1379
msgid "This problem was already reported to developers. Thank you!"
msgstr ""
"Fhuair an luchd-leasachaidh aithris air an duilgheadas seo mu thràth. Tapadh "
"leat!"

#: ../data/apportcheckresume.py:67
#: ../debian/tmp/usr/share/apport/apportcheckresume.py:67
msgid ""
"This occurred during a previous suspend, and prevented the system from "
"resuming properly."
msgstr ""

#: ../data/apportcheckresume.py:69
#: ../debian/tmp/usr/share/apport/apportcheckresume.py:69
msgid ""
"This occurred during a previous hibernation, and prevented the system from "
"resuming properly."
msgstr ""

#: ../data/apportcheckresume.py:74
#: ../debian/tmp/usr/share/apport/apportcheckresume.py:74
msgid ""
"The resume processing hung very near the end and will have appeared to have "
"completed normally."
msgstr ""
"Lean sinn air ach thuislich e aig an fhìor-dheireadh agus bidh coltas gun "
"deach a choileanadh mar bu chòir."

#: ../debian/tmp/usr/bin/apport-cli.py:74 ../bin/apport-cli.py:74
msgid "Press any key to continue..."
msgstr "Brùth putan sam bith airson leantainn air adhart..."

#: ../debian/tmp/usr/bin/apport-cli.py:81 ../bin/apport-cli.py:81
msgid "What would you like to do? Your options are:"
msgstr "Dè bu chaomh leat dèanamh? Seo na roghainnean agad:"

#: ../debian/tmp/usr/bin/apport-cli.py:85 ../bin/apport-cli.py:85
#, python-format
msgid "Please choose (%s):"
msgstr "Dèan taghadh (%s):"

#: ../debian/tmp/usr/bin/apport-cli.py:148 ../bin/apport-cli.py:148
#, python-format
msgid "(%i bytes)"
msgstr ""

#: ../debian/tmp/usr/bin/apport-cli.py:179 ../bin/apport-cli.py:179
msgid ""
"After the problem report has been sent, please fill out the form in the\n"
"automatically opened web browser."
msgstr ""
"An dèidh dhut an aithris air an duilgheadas a chur, nach lìon thu am foirm\n"
"ann an uinneag a' bhrabhsair a thèid fhosgladh gu fèin-obrachail?"

#: ../debian/tmp/usr/bin/apport-cli.py:182 ../bin/apport-cli.py:182
#, python-format
msgid "&Send report (%s)"
msgstr "&Cuir an aithris (%s)"

#: ../debian/tmp/usr/bin/apport-cli.py:186 ../bin/apport-cli.py:186
msgid "&Examine locally"
msgstr "&Sgrùd gu h-ionadail"

#: ../debian/tmp/usr/bin/apport-cli.py:190 ../bin/apport-cli.py:190
msgid "&View report"
msgstr "&Faic an aithris"

#: ../debian/tmp/usr/bin/apport-cli.py:191 ../bin/apport-cli.py:191
msgid "&Keep report file for sending later or copying to somewhere else"
msgstr ""
"&Glèidh faidhle na h-aithrise gus a chur uaireigin eile no airson lethbhreac "
"dheth a chur am badeigin eile"

#: ../debian/tmp/usr/bin/apport-cli.py:192 ../bin/apport-cli.py:192
msgid "Cancel and &ignore future crashes of this program version"
msgstr ""
"Sguir dheth agus le&ig seachad tuislidhean de thionndadh seo a' phrògraim "
"seo san àm ri teachd"

#: ../debian/tmp/usr/bin/apport-cli.py:194
#: ../debian/tmp/usr/bin/apport-cli.py:271
#: ../debian/tmp/usr/bin/apport-cli.py:303
#: ../debian/tmp/usr/bin/apport-cli.py:324 ../bin/apport-cli.py:194
#: ../bin/apport-cli.py:271 ../bin/apport-cli.py:303 ../bin/apport-cli.py:324
msgid "&Cancel"
msgstr "&Sguir dheth"

#: ../debian/tmp/usr/bin/apport-cli.py:222 ../bin/apport-cli.py:222
msgid "Problem report file:"
msgstr "Faidhle aithris an duilgheadais:"

#: ../debian/tmp/usr/bin/apport-cli.py:228
#: ../debian/tmp/usr/bin/apport-cli.py:233 ../bin/apport-cli.py:228
#: ../bin/apport-cli.py:233
msgid "&Confirm"
msgstr ""

#: ../debian/tmp/usr/bin/apport-cli.py:232 ../bin/apport-cli.py:232
#, python-format
msgid "Error: %s"
msgstr ""

#: ../debian/tmp/usr/bin/apport-cli.py:239 ../bin/apport-cli.py:239
msgid ""
"The collected information can be sent to the developers to improve the\n"
"application. This might take a few minutes."
msgstr ""
"'S urrainn dhut am fiosrachadh a chruinnich sinn a chur gun luchd-"
"leasachaidh ach\n"
"an urrainn dhaibh an aplacaid a leasachadh. Faodaidh gun doir seo beagan "
"mhionaidean."

#: ../debian/tmp/usr/bin/apport-cli.py:252 ../bin/apport-cli.py:252
msgid ""
"The collected information is being sent to the bug tracking system.\n"
"This might take a few minutes."
msgstr ""
"Tha am fiosrachadh a chruinnich sinn 'ga chur gu siostam tracadh nam "
"bugaichean.\n"
"Dh'fhaoidte gun doir seo beagan mhionaidean."

#: ../debian/tmp/usr/bin/apport-cli.py:302 ../bin/apport-cli.py:302
msgid "&Done"
msgstr "&Deiseil"

#: ../debian/tmp/usr/bin/apport-cli.py:308 ../bin/apport-cli.py:308
msgid "none"
msgstr "chan eil gin"

#: ../debian/tmp/usr/bin/apport-cli.py:309 ../bin/apport-cli.py:309
#, python-format
msgid "Selected: %s. Multiple choices:"
msgstr "Thagh thu: %s. Iomadh roghainn:"

#: ../debian/tmp/usr/bin/apport-cli.py:325 ../bin/apport-cli.py:325
msgid "Choices:"
msgstr ""

#: ../debian/tmp/usr/bin/apport-cli.py:339 ../bin/apport-cli.py:339
msgid "Path to file (Enter to cancel):"
msgstr ""
"An t-slighe a dh'ionnsaigh an fhaidhle (Brùth Enter airson sgur dheth):"

#: ../debian/tmp/usr/bin/apport-cli.py:345 ../bin/apport-cli.py:345
msgid "File does not exist."
msgstr "Chan eil am faidhle ann."

#: ../debian/tmp/usr/bin/apport-cli.py:347 ../bin/apport-cli.py:347
msgid "This is a directory."
msgstr "'S e pasgan a tha seo."

#: ../debian/tmp/usr/bin/apport-cli.py:353 ../bin/apport-cli.py:353
msgid "To continue, you must visit the following URL:"
msgstr ""
"Feumaidh tu tadhal air an URL a leanas mus urrainn dhut leantainn air adhart:"

#: ../debian/tmp/usr/bin/apport-cli.py:355 ../bin/apport-cli.py:355
msgid ""
"You can launch a browser now, or copy this URL into a browser on another "
"computer."
msgstr ""
"'S urrainn dhut brabhsair a chur gu dol an-dràsta no lethbhreac dhen URL seo "
"a chur ann am brabhsair no coimpiutair eile."

#: ../debian/tmp/usr/bin/apport-cli.py:357 ../bin/apport-cli.py:357
msgid "Launch a browser now"
msgstr "Cuir brabhsair gu dol an-dràsta"

#: ../debian/tmp/usr/bin/apport-cli.py:371 ../bin/apport-cli.py:371
msgid "No pending crash reports. Try --help for more information."
msgstr ""
"Chan eil aithris sam bith air tuisleadh ri dhèiligeadh. Feuch --help airson "
"barrachd fiosrachaidh."

#~ msgid ""
#~ "This occured during a previous suspend and prevented it from resuming "
#~ "properly."
#~ msgstr ""
#~ "Thachair seo nuair a chaidh a chur 'na dhàil roimhe agus cha b' urrainn "
#~ "dhuinn leantainn air cùisean mar bu chòir."

#~ msgid ""
#~ "This occured during a previous hibernate and prevented it from resuming "
#~ "properly."
#~ msgstr ""
#~ "Thachair seo nuair a chaidh a chur 'na chadal-geamhraidh roimhe agus cha b' "
#~ "urrainn dhuinn leantainn air cùisean mar bu chòir."

#~ msgid "%prog [options] <apport problem report | crash ID>"
#~ msgstr "%prog [options] <apport problem report | crash ID>"

#~ msgid "incorrect number of arguments; use --help for a short help"
#~ msgstr ""
#~ "àireamh chearr de dh'argamaidean; cleachd --help airson beagan cobharach"

#~ msgid ""
#~ "you either need to do a local operation (-s, -g, -o) or supply an "
#~ "authentication file (--auth); see --help for a short help"
#~ msgstr ""
#~ "feumaidh tu a dhèanamh gu h-ionadail (-s, -g -o) no feumaidh tu faidhle "
#~ "dearbhaidh a shònrachadh (--auth); faic --help airson beagan cobharach"

#~ msgid "Your system might become unstablenow and might need to be restarted."
#~ msgstr ""
#~ "Dh'fhaoidte gum fàs an siostam agad neo-sheasmhach a-nis agus gum bi agad ri "
#~ "ath-thòiseachadh."
